tests/test_motifs.py

```diff
@@ -301,3 +368,67 @@ def test_match_mean_stddev_isconstant(Q, T):
     )
 
     npt.assert_almost_equal(left, right)
+
+
+def test_motif_random_input():
```

Comment:
I think the name of this func should be changed. However, I do not know  have any suggestion.

Reply:
What is the point of testing random input? If there is none, then why not simpy `test_motifs` (unless it's already taken)?

Reply:
I wanted to reflect the differences between test functions in their names. You are right..."why not just `test_motids`?" :) 

I am going to use `test_motifs`